src/Parser/Field/FieldFactory.php

```diff
@@ -51,7 +51,7 @@ public static function build(
             } else {
                 $data = $item->get($field);
                 if (isset($metadata['fields'])) {
-                    if (isset($metadata['fields'][$field])) {
+                    if (isset($metadata['fields'][$field]) && isset($metadata['fields'][$field]['data']['type'])) {
```

Comment:
Weird … I would have thought that would translate to an exception in 3.2.x as well … Must have been off my game during that cycle :stuck_out_tongue_winking_eye: